flow/flow.go

```diff
@@ -69,22 +69,16 @@ const (
 	FlowStateConsentError = int16(129)
 )
 
-// Flow is an abstraction used in the persistence layer to unify LoginRequest,
-// HandledLoginRequest, ConsentRequest, and AcceptOAuth2ConsentRequest.
-//
-// TODO: Deprecate the structs that are made obsolete by the Flow concept.
-// Context: Before Flow was introduced, the API and the database used the same
-// structs, LoginRequest and HandledLoginRequest. These two tables and structs
-// were merged into a new concept, Flow, in order to optimize the persistence
-// layer. We currently limit the use of Flow to the persistence layer and keep
-// using the original structs in the API in order to minimize the impact of the
-// database refactoring on the API.
+// Flow is an abstraction used in the persistence layer.
+// It has outlived its usefulness but is difficult to remove.
```

Comment:
I think the explanation before was better to understand for anyone looking to still address this. The current code doc is not clear what "outlived it's usefulness" means. Please revert!

Reply:
reverted